### ISSUE TITLE
Atziri's Reign Skills Disappear When Manipulating Items With Unleash Support Socketed

### ISSUE THREAD

**Issue by hd1867** [PathOfBuildingCommunity/PathOfBuilding#5952]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### How is Path of Building expected to behave?

Under normal circumstances, regardless of the support gems socketed, the Atziri's Reign staff adds 3 skills to your character, and those skills shouldn't disappear when changing items that are not your weapon. 

### How does Path of Building behave?

When making any change in the Items window with an unleash support socketed in Weapon 1, all of the skills added by Atziri's Reign Disappear. 

Changes can include: 
- Crafting an item
- Saving after editing a crafted item
- Adding a unique item
- Searching for a unique item
- Saving after editing the rolls of a unique item

### How to reproduce the issue

1. Add Atziri's Reign to a fresh PoB 
2. Add an unleash support and set "Socketed In" to weapon 1
3. Set the active skill to any skill provided by Atziri's Reign
4. Add any other item to the build 
5. You should see the active skill is now empty

### Character build code

```shell
eNrtHG1z4rj58_IrNMxcp51eAuYtbJrcDQGS0AlZFpJt--lG2AJ0K1usLSdhb-6_95Fkg0OQsWN3pp3e3syuQc-7Hj1vyHfx84vL0BPxA8q9y6p1Wq8i4tncod7ysvr4cH3Srf78U-VigsXq0-IqpEyu_FT5cKGeESNPhAFeFQnsL4n4ElNq_gKU1tgTK8K9Mf6V-zfcuazec49U0Rx7DhXxJ5vhILjHLrmszmxAriIc2MRz-rvvI8AV9rEtiH8nufZCwcfcgVXhh7DqYurNuP2ViBufh2sQoYqeKHnWMKPx5NP0ISES9ZIigUYfLiYMb4g_E1igAP66rPbAMHhJbqkAUpiFQKfRbDROO-1qLRXjKvQDMcAuPGbFnK0JcbbA9dPu2dlZo949q3fOup2GEYv4wPHBp8sl8adYkAQFI6uJT4aLBbEFfSJ9n4r-Cnv2DrNjwssLOw6ZoGtGib-Ft8xS3b4hbtXrJuAHLjAbTGY769ab3dNOs9vs1htd6ywdj-_2s34MMsnECNwPGYNzkQl2SgLYNCzA07PR5u6cesTpPS3zatvnnDn82Uua3-qaoMfYw30eZLCNhJwQHw6pyIUwIzaHc52XR07MO7og2SFz6REh5JXmfXoMZ1nhchN-n0BTCGfZIGc8ZBkhhZ_wavOZ-ZYEtKymCXJAXjLQG3kiG9ckYCrXJy5U6soUVoa3k10EbZ92ular1fr40Tr7aAzZq01AbczG-IW6oQux8gF_JTt-nRT_W66EB8HJhNo0BoRr6pP8WH2IOe_AWmEe5EeTpyuDESC52ucSduTZ2Y7so-erOJ3IyZ1UhCkcJJn854xkxNixiI5jlsSnWS2JF_HbZFPnjhB7dQMV0usaIS34bqHa6WaVsJnMKgEPmLWdESGHkSSiwUinH9OQcppp6BF_uZmtKGFOPuhYsD5eZ4iR0sxJ7Ezmfs0ul8ckUXOa5B_Yd7JlkrwyPeEgGWGtTrq5NHg2xySMEEBwyF4BWjdX0PxXWT-zfGg93-Whn3HDNXAmBeLsoDuOKXFCO1s2umLQMGWVHqRi7BCGOYP2hMD21wF3liQXl_wYs3C9hkgh9zwrnkxvUIzTRJly0skA_QkcNtO5lZkwO4MddGYG29yencseSnZdZH7OocwOPDOLbU86hpDgQqhXLTG077szb9wc6L8yNVMKMGNTN-HPIPlKzjWCfNBQx-xqF6MoPvG-bzLTfwWeicHQc0JfHoXMPPYxDrF5oC6EyyAYYIGRE9W9X7BPsScaahIUEOzbqzvY-mvM2BwCwGU1-a38tIdoxXt7UVNzJfk0ctfcF-rLPmZ2oLiPvHUokKeGQi4N7F_m4WIhJzzAQPhqaDW8vh72H0ZfhpHASZTgK2XsFy9053Imof_dudWMqLCJgnAe6MfL6hdKnmcSa0AEpgyMaHPG8DogzmV1gVkAnCk8KpgZWMYWKdQUFJS38WjoMK0dgJnS8IX4AvSFfGv7lBjl2q4fEUozlLlYbrCJmpycmAnpaN_HgdC1gsFSatBlpiInR0Z15GIKLvgMZkbO0eoRS4jNmgSQgOmC2vJspG_5A0BrqBS72DYcK3uTst9RzjLTuGJqPHiYgF40I084DWQ5cBg7Wk2x6tKj5v2MVs3oA2Jjo-560Yy8LT-5p8awh6lsoVIo3XNPOTkcmh5lMr8Yd3bIyBbETPCTWBFfJy0jpTHEqBgk9eD4dB4K8zFOQKTYSjXDBgvJNTOqbvgMOsi1lEj0qgkyGDQJYyalmwdjIEtD1bWG0X5R5ZKyBVFpbjC_Xk0xQtydGPSPllMOiYq_vSdOHV29Go7LHlhawIBCvTgZVYkXJ7NfoxeneM1w8NW439GqGf0RmjcqNgeo6F-TMhGRh6oYBXm2ilGY7hcSO9xpegmh4teALAgYOjWAbWFS9leE3gCSnEjZ24yklFiHD8JOtVy0dCg_qGluitpBo_Fkmg9rkCOEIBfdplQ72Sht-6ZbgplYTThnxQi-GcMW0pOLAHvOQE57Cioqh0XhGojFkn06VK7utnSf6kUtbiQuVO0WoAA6jBviBlcb6AivZXba_d6mekSsDBsV90K3N0mcmJ1DFjhk8vvPIZaBRXZZiW_v9O_kHvfd7RAfSK34swyKmqIsJQPINXd3eqXHRERMsovtpa0TCaTUtNR30ZfqZ_DeTmqlsZKbejYLHTLyomYZpKHgKAzPpWRSr9C35S_lgrjn1nlPfKc-_RNe8-BvAZqGjPyI_h5CMpBnCUHJu1jIqwJy3r01esC4HAESQPKQVd2XZSvqhwvQKEK-YXyOWSOWRtupUa--Wra2HCSdETD8HBLwpwFxwR2q6FvC6Hsygct4IqIum0A5W4vQY-UiKjWzbJpUZuGiPb0FqjZ0Qz1PUJuuo-AYyTpSxa_yjyoCq8rPY-gxHeira3IPgprc9trOP1LJpahuJRV_9BjBwSpS9qKmNua4_yjz_Td4kORYlg9pAR89-i0kShxIti6ZQ8YV73AphRwgvtgp-YdL_S-4lPWfc6iZgIAP-rJ3-JPC_cOdSnWncqPN_4HNo0coNVT9pMsm-fjgExJXR4qEFVUm8CF5f08qUJeWDQTxN9EoRG6Fxx1Z7bS73WazigSQS9wdbFjRtUCNrqV6nN7pvVgJsQ7Oa7Xn5-dTUHjFF-SFMnJqc7e2BiSQ6URZ_USSrfXgz9Wyp_5o9WJKF_q-YFCLVAXZlZpSOflwzwVRc2f5ZfxBqS7HwtFIeyZ8qc93zt1_KV1l7QaluBjjdeR_ci2qAJtR1SewGFCwhq8ajnhvJOA_4wH9hdrvyMjyOa5Aw4Doazvac9XXsacqpGRZOMU-OMI5erwffX4cVt5G0MpeAK1ERec5atQrkYHO0dXJ9r-KUmVKvp2jTrcyctcMHE_CWJW_Nuo_ID3LRIIjNYNAqvdHuhhHzyvYKfQsS21ofhCOeN742BMBUpSBLzpQnyHlfJVDKzb2UHQHckfiQD7Ojvw2-Fb62PO4QHOCZLMLDUIlUsneLkzJQv4AAEvSucbcmWIPAHz5t76RGe1MdM6MEI2jEM2jEK2jEPFVTBVYXjtP47XzTHvTYeUeXF4CVK5YSNAEnJ-hngseBLbx8QLUPkfSjSsTnyzoyzmSN2tTPszCRZYPW3esJxzv7Oy14_0m8DI494muF36EjcW__6b0PQd1f_9zq3vS7vzlB5kgfIh0xEFy8oLUHQSixxZI_miRaeP2DCb7MNORfN2gQbpBFDAmc4hAKgiqX6UgYTzJuYRc0gGvlo6wy1gxinUM5YpDH456800QYIb0sUbtHDyvCBP7-I0c-GqQhZrv0BLNnvF6n3XrPYQKqB_RaJRA47BCnRIU6pSg0LtoHFLIyuNch7yzWYJFGmVpk8ffbglziShkjwOHrVNYgkZuCoVZNsuyf7sEZ7BKOB5WCXK0cnpCWSGmUfREtkowYP6cUVqIbRbVv1HwCOcTwNkg_btjER_UtUURCocPdrswhU5Z22qVFWPyu2ZxO7YK72W7HKfKo7yu-kuIhM0SAkqz4KFsFcS3yrH_uzNlbqdt5cZ4TxVV0KrtorG6pG1plRVcSqur391C5cCbytlMo3BoapSzB6VlijIavtb7dMpr_OI5vqRio11YkOIprpM3GBQNHm8Z6plPPAdXk1l1M5l7C7p8c894_8X33e1k9bb524vJ8eRK3mF8fZ2592U47d0cusy8_yr7USbEI-5mFFzxIMgGO6PfyUS-fyV28GPi0NA9gKFf6j9IeMKwTVacOcR_RV6_8h5fwz5LvtB1CF5e_o1vTsdIzSM48emL4RuN4zziF0hinHY6SvTWDWdMTQ2TCqUjbi8Nxgids3SE1-_5JBjF11sib7yo7f__Gv4N80IhMw==
```

**Comment by Paliak** [comment]
While i've not been able to reproduce the issue, this is likely due to how unleash currently interacts with trigger skills. Should be fixed in #4599 .

**Comment by QuickStick123** [comment]
I believe this is fixed as I can't reproduce it anymore and 4599 has been merged.
